.ado/jobs/build-test-rntester.yml

```diff
@@ -89,88 +89,77 @@ parameters:
 
 jobs:
   - ${{ each slice in parameters.appleBuildMatrix }}:
-      - job: ${{ slice.name }}
-        displayName: ${{ slice.friendly_name }}
-        pool:
-          vmImage: $(vmImageApple)
-        timeoutInMinutes: 90
-        cancelTimeoutInMinutes: 5
-        steps:
+    - job: ${{ slice.name }}
+      displayName: ${{ slice.friendly_name }}
+      pool:
+        vmImage: $(vmImageApple)
+      timeoutInMinutes: 90
+      cancelTimeoutInMinutes: 5
+      steps:
         - template: /.ado/templates/apple-tools-setup.yml@self
 
         - ${{ if in(slice.sdk, 'xros', 'xrsimulator') }}:
-            - task: CmdLine@2
-              displayName: Download visionOS SDDK
-              inputs:
-                script: |
-                  set -eox pipefail
-                  # https://github.com/actions/runner-images/issues/10559
-                  sudo xcodebuild -runFirstLaunch
-                  sudo xcrun simctl list
-                  sudo xcodebuild -downloadPlatform visionOS
-                  sudo xcodebuild -runFirstLaunch
-
-        - task: CmdLine@2
-          displayName: yarn install
-          inputs:
-            script: |
+          - script: |
```

Comment:
Prefer script to cmdline because the former is x-plat I assume?

Reply:
`script` is short-hand for cmdline. Just less to write and more concise in general.